src/render.c

```diff
@@ -71,7 +71,9 @@ void render_con(Con *con, bool render_fullscreen, bool already_inset) {
             }
         }
         inset.height = -inset.y;
```

Comment:
@michaeladler Can you see if i3-easyfocus also works if, instead of introducing the `if` here, you change the previous line to `inset.height = 0`? Because that's the other fix I was talking to you about that I already know of and reading i3-easyfocus' code, I think it should also fix your issue.